src/baldr/edgeinfo.cc

```diff
@@ -93,9 +93,41 @@ std::vector<std::string> EdgeInfo::GetNames() const {
       throw std::runtime_error("GetNames: offset exceeds size of text list");
     }
   }
+  names.shrink_to_fit();
```

Comment:
this has performance implications, we should revert it as that type of thing isnt in the scope of this pr anyway